src/nxt_router.c

```diff
@@ -2172,6 +2194,29 @@ nxt_router_conf_create(nxt_task_t *task, nxt_router_temp_conf_t *tmcf,
 
 #endif
 
+#if (NXT_HAVE_OTEL)
+    otel          = nxt_conf_get_path(root, &telemetry_path);
+    otel_endpoint = nxt_conf_get_path(root, &telemetry_endpoint_path);
+    otel_batching = nxt_conf_get_path(root, &telemetry_batch_path);
+    otel_sampling = nxt_conf_get_path(root, &telemetry_sample_path);
+    otel_proto    = nxt_conf_get_path(root, &telemetry_proto_path);
+
+    if (otel) {
+        nxt_conf_get_string(otel_endpoint, &telemetry_endpoint);
+        nxt_conf_get_string(otel_proto, &telemetry_proto);
+        telemetry_batching = otel_batching ? nxt_conf_get_number(otel_batching) : NXT_OTEL_BATCH_DEFAULT;
+        telemetry_sample_fraction = otel_sampling ? nxt_conf_get_number(otel_sampling) : NXT_OTEL_SAMPLING_DEFAULT;
+
+        nxt_otel_rs_init(&nxt_otel_log_callback,
+                      &telemetry_endpoint,
+                      &telemetry_proto,
+                      telemetry_sample_fraction,
+                      telemetry_batching);
+    } else {
+      nxt_otel_rs_uninit();
```

Comment:
Should we be calling this if we didn't do the above `_init()`?

Reply:
yes its safe. the uninit code is [here](https://github.com/javorszky/unit/blob/dac737c8501edd0608363f7fbcb139f0d1487c45/src/otel/src/lib.rs#L54)

You can see it will pass through acceptably. 
Furthermore, this is needed in case the old configuration has an initialized tracer and the new one does not include a config for it.